### DISCUSSION TITLE
Δεν στέλνουμε μαζεμένες ερωτήσεις σε ένα θέμα!

### DISCUSSION THREAD

**Discussion by mariosconsta** [courses-ionio/hci#1600] [Announcements]
“Οι πρώτες εβδομάδες και τα πρώτα παραδοτέα αυτής της εργασίας μέχρι την 6η εβδομάδα είναι αναφορές με τις ενέργειες που έγιναν για την εγκατάσταση και προσαρμογή του νέου λειτουργικού συστήματος, π.χ., την 1η εβδομάδα θα πρέπει να εντοπίσετε τα χαρακτηριστικά που έχει το υλικό σας (δίσκος, μνήμη, επεξεργαστής, δίκτυο, γραφικά, κτλ)” </br>

Δηλαδή οι πρώτες 3 ασκήσεις γραμμής εντολών μπορούν να είναι της κατηγορίας “Warm Up”;</br>

Παράλληλα, έχω δώσει το μάθημα τον Σεπτέμβρη, θα μπορούσα τις ασκήσεις γραμμής εντολών που έκανα τότε να τις κρατήσω για την εξεταστική που έρχεται και να ασχοληθώ με το συμμετοχικό υλικό που ήταν και ο λόγος που δεν πέρασα το μάθημα; Θα μπορούσε κάποιος να δει τις ασκήσεις γραμμής εντολών στο branch μου (2015002) και να μου επιβεβαιώσει ότι όντος αυτά είναι τα ζητούμενα και ότι είναι αποδεκτές εφόσον δεν μπορώ να τις παρουσιάσω από κοντά;</br>

Επίσης με τις ασκήσεις γραμμής εντολών και το συμμετοχικό υλικό μπορώ να περάσω με 5 εφόσον δεν μπορώ να παρευρεθώ στο νησί για παρουσιάσεις;</br>

Τέλος έχω χαθεί λίγο με το τι πρέπει να κάνω στο συμμετοχικό υλικό. Πιστεύω έκανα σωστά μόνο το Α1 τον Σεπτέμβρη αλλά δεν είμαι σίγουρος. Έχω κοιτάξει το [λινκ ](https://courses-ionio.github.io/projects/social/)αλλά όπως σας είπα μπερδεύτηκα αρκετά.</br>

Ευχαριστώ εκ τον προτέρων!

**Comment by epidrome** [comment]
https://github.com/courses-ionio/hci/discussions/1601
